product_profile_example/__manifest__.py

```diff
@@ -1,24 +1,15 @@
-# coding: utf-8
 # © 2015 David BEAL @ Akretion
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'Product Profile Example',
-    'summary': 'Product Profile Use Case',
-    'version': '10.0.1.0.1',
-    'author': 'Akretion, Odoo Community Association (OCA)',
-    'category': 'product',
-    'depends': [
-        'product_profile',
-        'purchase',
-        'point_of_sale',
-        'mrp',
-    ],
-    'website': 'http://www.akretion.com/',
-    'demo': [
-        'product.profile.csv',
-        'product_product_demo.xml',
-    ],
-    'installable': True,
-    'license': 'AGPL-3',
+    "name": "Product Profile Example",
+    "summary": "Product Profile Use Case",
+    "version": "12.0.1.0.1",
```

Comment:
`12.0.1.0.0`

Reply:
Hi Nikul, removing the license triggers the pylint error:
[C8102(manifest-required-key), ] Missing required key "license" in manifest file
or is there something that I'm misunderstanding ?